.gitattributes

```diff
@@ -0,0 +1,9 @@
+################
+# Line endings #
+################
+*     text=auto
+
+###################
+# GitHub Linguist #
+###################
+*.qmd linguist-language=Markdown
```

Comment:
It seemed to me like they were already getting Markdown syntax highlighting. Was there something in particular that prompted this?